be/src/exec/pipeline/runtime_filter_types.h

```diff
@@ -220,14 +220,16 @@ class PartialRuntimeFilterMerger {
     // mark runtime_filter as always true.
     bool set_always_true() {
         _always_true = true;
-        return _try_do_merge({});
+        (void)_try_do_merge({});
```

Comment:
is it safe to ignore error status returned by _try_to_merge?

Reply:
yes